frontend/src/constants/routes.ts

```diff
@@ -23,5 +25,5 @@ export const PATHS: {
 export const API: {
   getProfile: string;
 } = {
-  getProfile: '/api/account/profile',
+  getProfile: `/api/users/${userId}`,
```

Comment:
Are you sure that we should replace this route?

Reply:
I suppose, In present we will work with this